addons/account/wizard/account_payment_register.py

```diff
@@ -56,18 +57,28 @@ class AccountPaymentRegister(models.TransientModel):
         compute="_compute_qr_code",
     )
 
-    # == Fields given to detect if the user change the date after he modifies the amount of the payment == #
-    wizard_change_mode = fields.Selection(
+    batches = fields.Binary(compute='_compute_batches', export_string_translation=False)
+    installments_mode = fields.Selection(
         selection=[
-            ('compute_amount_change', 'Amount recomputed'),
-            ('payment_date_change', 'Payment date change'),
+            ('next', "Next Installment"),
+            ('overdue', "Overdue Amount"),
+            ('full', "Full Amount"),
         ],
-        export_string_translation=False,
+        compute='_compute_installments_mode',
+        readonly=False,
+        store=True,
+        export_string_translation=False
     )
-    # Specify if the user changed the amount or not based on 'wizard_change_mode'
-    is_amount_forced_by_user = fields.Boolean(
-        export_string_translation=False,
+    installments_switch_html = fields.Html(
+        compute='_compute_installments_switch_values',
+        sanitize=False,
```

Comment:
@odoo/rd-security Unsanitized computed html field added. The html is safe. Can you check it plz?

Reply:
Though I agree the computed HTML looks safe, can you explain why it must not be sanitized at all ?

Reply:
@beledouxdenis We have a custom owl component for this field that handle a click on an hightlighed text. I though I had to make the field `sanitize=False` to make it works (that's we did on the bank reconciliation wizard for the same reason) but @dylankiss told me it's not necessary...

Reply:
@beledouxdenis It seems replacing the `div` to `span` solves the need to `sanitize=False`. @dylankiss updated the code and removed the sanitize

Reply:
@beledouxdenis Switching the `<div>` to a `<span>` made the sanitize work fine, so it's removed now ;)

Reply:
Thanks for having a look :)